inst/extdata/config.R

```diff
@@ -18,7 +18,13 @@ CONFIG$MD_TABLE_COLUMN_WIDTHS <- list(
   )
 )
 
-CONFIG$REGISTER_COLUMNS <- list("Certificate", "Repository", "Type", "Venue", "Issue", "Report", "Check date")
+# These are the columns to keep in the register table
+CONFIG$REGISTER_COLUMNS <- list(
+  other = c("Certificate", "Paper Title", "Type", "Venue", "Issue", "Report", "Check date"),
```

Comment:
"other" is basically HTML and MD then, right?

If other is capturing only these two cases, I would prefer a more explicit approach and list both here. OK for you? Am I missing something?

Reply:
Yes "other" was just for md and html. I implemented the change